test/TestProjects/FirstTest-Cadl/FirstTest-Cadl.cadl

```diff
@@ -133,6 +138,12 @@ op patchAction(@body body: Thing): Thing;
 @convenienceAPI
 op anonymousBody( ...Thing ): Thing;
 
+@route("/aliasBody")
+@doc("body parameter without body decorator")
+@post
+@convenienceAPI
+op aliasBody( ...ThingAlias ): Thing;
+
```

Comment:
One question related to current implementation:

If we have 
op alias2Body( ...ThingAlias ): Thing;
op alias3Body( ...ThingAlias ): Thing;

It seems we will create Alias2BodyRequest，  Alias3BodyRequest with all their derived classes.
Do we have a way to combine them as all of them are mapping to '...ThingAlias'

Reply:
That depends on how emitter will output the body parameter for this scenario.
And it does not matter because those models for aliases will always be internal.